news/views.py

```diff
@@ -87,6 +87,8 @@ def get_context_data(self, **kwargs):
         context.update({
             'article': event,
             'timeplaces': event.timeplace_set.all() if event.standalone else event.timeplace_set.future(),
+            'is_old': not event.timeplace_set.future().exists(),
+            'last_occurrence': event.get_past_occurrences().exists() and event.get_past_occurrences().first() or None
```

Comment:
This way of writing a conditional assignment is pretty hard to read; please replace it with
```python
event.get_past_occurrences().first() if event.get_past_occurrences().exists() else None
```

Reply:
This is just me not thinking. Actually, `first()` returns `None` if there are no objects in the queryset, so no conditional is even needed.